examples/scenes/src/test_scenes.rs

```diff
@@ -97,6 +101,11 @@ fn cardioid_and_friends(sb: &mut SceneBuilder, _: &mut SceneParams) {
 }
 
 fn animated_text(sb: &mut SceneBuilder, params: &mut SceneParams) {
+    let piet_logo = params
+        .images
+        .from_bytes(PIET_LOGO_IMAGE.as_ptr() as usize, PIET_LOGO_IMAGE)
```

Comment:
This is just my expedient hacky solution to support image loading for examples. Being an example though, it probably does deserve a “don’t do this in real code!” comment.

Since the address is never converted back into a pointer, I don’t think provenance is an issue. Size likely would be on CHERI architectures though.

Reply:
Fair enough! Pointer to int casts are always worth calling out in code review just to have some discussion logged for future reference.